apis/r/R/write_soma.R

```diff
@@ -351,7 +351,7 @@ write_soma.TsparseMatrix <- function(
 ) {
   check_package('SeuratObject', version = .MINIMUM_SEURAT_VERSION())
   stopifnot(
-    "'x' must be a data frame" = is.data.frame(x),
+    "'x' must be a data frame" = is.data.frame(x) || inherits(x, 'DataFrame'),
```

Comment:
Wouldn't `inherits(x, "DataFrame")` cover all the cases we are interested in?

Reply:
No; `base::data.frame` and `S4Vectors::DataFrame` are two indepent classes (albeit ones that function nearly-identically) and must be checked separately

Reply:
Oh I always fall for `S4Vectors::DataFrame` and forget this.  Thanks, and I withdraw this as resolved.